spec/food_spec.rb

```diff
@@ -0,0 +1,45 @@
+# spec/models/food_spec.rb
```

Comment:
**Your code looks amazing perfect implementation!**

- [ ] However, In this project make sure you have written clean code. Kindly remove unused commented code.

Reply:
[OPTIONAL] Please personalize your MIT license.